tests/dependency-tests/root.rs

```diff
@@ -0,0 +1,285 @@
+use std::process::Command;
+
+#[derive(Debug)]
+enum Search<'a> {
+    Positive(&'a str),
+    Negative(&'a str),
+}
+
+#[derive(Debug)]
+struct Requirement<'a> {
+    human_readable_name: &'a str,
+    target: &'a str,
+    packages: &'a [&'a str],
+    features: &'a [&'a str],
+    default_features: bool,
+    search_terms: &'a [Search<'a>],
+}
+
+const ALL_WGPU_FEATURES: &[&str] = &[
```

Comment:
suggestion: @cwfitzgerald indicated that the intent of this ordering is to match our `Cargo.toml` declaration order of these features. Can we get a `NOTE` comment here and there for that, please?